index.html

```diff
@@ -40,6 +51,7 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
           <h2>My Recent Works</h2>
           <hr>
         </hxr>
+
         <section class="multi-stories ">
           <img src="assests/images/img_placeholder.png" alt="featured image">
   
```

Comment:
- [ ] Please align the first section of the image with the second section of the image on the same line. If you see the Figma design then you find the image on the same line. See the image below for reference
- [ ] Please use the border for the second image as well. that is: `professional image part`
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/199895373-f0418fbb-df17-432e-851b-5df660e310e4.png)
### Your design
![his](https://user-images.githubusercontent.com/81727988/199895433-92950c82-cbe8-4ee8-a3ce-25903377d6c7.png)